build/docker/Dockerfile

```diff
@@ -39,15 +39,10 @@ RUN chmod +x /usr/local/bin/docker-run.sh
 
 RUN pecl install xdebug && docker-php-ext-enable xdebug
 RUN echo 'zend_extension="/usr/local/lib/php/extensions/no-debug-non-zts-20180731/xdebug.so"' >> ${PHP_INI_DIR}/php.ini
-RUN echo 'xdebug.remote_autostart=1' >> ${PHP_INI_DIR}/php.ini
-RUN echo 'xdebug.remote_enable=1' >> ${PHP_INI_DIR}/php.ini
-RUN echo 'xdebug.default_enable=1' >> ${PHP_INI_DIR}/php.ini
-#RUN echo 'xdebug.remote_host=docker.host' >> ${PHP_INI_DIR}/php.ini
-RUN echo 'xdebug.remote_port=9000' >> ${PHP_INI_DIR}/php.ini
-RUN echo 'xdebug.remote_connect_back=1' >> ${PHP_INI_DIR}/php.ini
-RUN echo 'xdebug.profiler_enable=0' >> ${PHP_INI_DIR}/php.ini
-RUN echo 'xdebug.remote_log="/tmp/xdebug.log"' >> ${PHP_INI_DIR}/php.ini
```

Comment:
Do we really need to remove this line (same question for other removed line) ?

Reply:
I just used the settings suggested by IDE. 

Now i've checked the following guide: https://xdebug.org/docs/upgrade_guide:
autostart =  start_with_request
connect_back = discover_client_host

The following three is not strictly needed:
1) I've left the host.docker.internal address for safety, in case discovery does not work.
2) I've commented out the xdebug.log because it is a really chatty log for debugging xdebug problems. 
3) Added idekey because it is needed for netbeans, but ignored by eclipse. 

I would keep these if it is ok with you (unless somebody has a problem with it).

Reply:
Thanks for feedback. We will make a try with your version.